model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -48,7 +48,12 @@ data class AnalyzerConfiguration(
      * Package manager specific configurations. The key needs to match the name of the package manager class, e.g.
      * "NuGet" for the NuGet package manager.
      */
-    val packageManagers: Map<String, PackageManagerConfiguration>? = null
+    val packageManagers: Map<String, PackageManagerConfiguration>? = null,
+
+    /**
+     * A flag to control whether excluded scopes and paths should be ignored during the analysis.
```

Comment:
Consider using "skipped" instead of "ignored", which hopefully makes more clear what we're not analyzing and discarding the results, but not analyzing at all.